inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -15,6 +15,15 @@ std::vector<std::string> disabledTestPatterns() {
         ".*ActivationLayerTest\\.CompareWithRefs/Log.*netPRC=FP32.*",
         ".*ActivationLayerTest\\.CompareWithRefs/Sigmoid.*netPRC=FP32.*",
         ".*ActivationLayerTest\\.CompareWithRefs/Relu.*netPRC=FP32.*",
+        // Not supported dynamic shapes without upper bound
```

Comment:
Just to clarify, are these tests now enabled, but not working?

Reply:
Yes